Dockerfile

```diff
@@ -17,19 +17,7 @@ COPY --from=license-check /license-check /usr/bin/
 RUN mkdir -p /go/src/github.com/inlets/inlets-operator
 WORKDIR /go/src/github.com/inlets/inlets-operator
 
-# Cache the download before continuing
-COPY go.mod go.mod
-COPY go.sum go.sum
-RUN go mod download
-
-COPY pkg  pkg
-COPY main.go  main.go
-COPY image_test.go  image_test.go
-COPY controller.go  controller.go
-COPY validate.go validate.go
-COPY validate_test.go validate_test.go
-COPY config.go  config.go
-COPY config_test.go  config_test.go
+COPY . .
```

Comment:
I didn't request this change and it prevents the caching that was put in place. Can you revert?

Reply:
It has the same caching behavior because the vendor folder is copied we don't need the go download.

Go will use the modules in the vendor folder.

Reply:
This change has been taken out of the PR.